pkg/operator/operators/klusterlet/controllers/klusterletcontroller/klusterlet_cleanup_controller.go

```diff
@@ -62,7 +60,6 @@ func NewKlusterletCleanupController(
 	}
 
 	return factory.New().WithSync(controller.sync).
-		WithInformersQueueKeyFunc(helpers.KlusterletSecretQueueKeyFunc(controller.klusterletLister), secretInformer.Informer()).
```

Comment:
in hosted mode, we need the external-managed-kubecnfig to build a client to clean up resources on the managed cluster. so I think it is necessary to watch this secret to trigger a cleanup reconcile.

Reply:
added back.